node/src/actors/chain_manager/mod.rs

```diff
@@ -909,8 +909,8 @@ impl ChainManager {
                         let mut rng = rand::thread_rng();
                         let checkpoints_period = act.consensus_constants().checkpoints_period;
                         let superblock_period = act.consensus_constants().superblock_period;
-                        let end_range = if superblock_period > 4 {
-                            (superblock_period - 3) * checkpoints_period
+                        let end_range = if superblock_period > 5 {
+                            (superblock_period - 5) * checkpoints_period
```

Comment:
There is a comment a few lines above this that still says

// Broadcast vote between one and ("superblock_period" - 3) epoch checkpoints later.

Reply:
According to the documentation, `rng.gen_range`

> Panics if low >= high.

So we need to ensure that `end_range` is `>` than `checkpoints_period`. After this change it can be equal, when `superblock_period = 6`.